java/java-jersey-jaxrs/src/main/java/io/swagger/sample/data/PetData.java

```diff
@@ -68,11 +69,13 @@ public Pet getPetById(long petId) {
 
```

Comment:
I missed the iterator around it. I also removed the synchronizedLists since all the operations are within sync {}.